Lib/test/test_tools/test_msgfmt.py

```diff
@@ -144,20 +154,70 @@ def test_nonexistent_file(self):
         assert_python_failure(msgfmt, 'nonexistent.po')
 
 
+class MultiInputTest(unittest.TestCase):
+    """Tests for multiple input files
+    """
+
+    def test_no_outputfile(self):
```

Comment:
This test looks redundant. `test_both_without_outputfile` supersedes it, and this is **Multi**InputTest.